ui/media/js/main.js

```diff
@@ -1146,37 +1031,18 @@ function onDimensionChange() {
 
 saveToDiskField.addEventListener('click', function(e) {
     diskPathField.disabled = !this.checked
-    handleBoolSettingChange(SAVE_TO_DISK_KEY)(e)
 })
 
 useUpscalingField.addEventListener('click', function(e) {
     upscaleModelField.disabled = !this.checked
-    handleBoolSettingChange(USE_UPSCALING_KEY)(e)
 })
 
-function setPanelOpen(panelHandle) {
-    let panelContents = panelHandle.nextElementSibling
-    panelHandle.classList.add('active')
-    panelContents.style.display = 'block'
-}
-
-if (isAdvancedPanelOpenEnabled()) {
-    setPanelOpen(advancedPanelHandle)
-}
-
-if (isModifiersPanelOpenEnabled()) {
-    setPanelOpen(modifiersPanelHandle)
-}
-
-if (isNegativePromptPanelOpenEnabled()) {
-    setPanelOpen(negativePromptPanelHandle)
-}
-
 makeImageBtn.addEventListener('click', makeImage)
 
 
 function updateGuidanceScale() {
     guidanceScaleField.value = guidanceScaleSlider.value / 10
+    guidanceScaleField.dispatchEvent(new Event("change"))
```

Comment:
I've fixed this temporarily in beta (didn't want to wait for this full PR to fix the bug). But I like your solution better, so when you're resubmitting this PR, please delete my fix.

I put `guidanceScaleSlider.dispatchEvent(new Event("change"))` in the next function `updateGuidanceScaleSlider()`, since the current auto-save code still listens to guidanceScaleSlider. And similarly for the prompt strength bit as well.

Please remove my fix before submitting yours, since yours is better.

Reply:
I couldn't find your change for this? Not sure if it got removed already or if I missed it.